### DISCUSSION TITLE
enoent ENOENT: no such file or directory, open '/vercel/path0/package.json'

### DISCUSSION THREAD

**Discussion by teddyrasmussen** [vercel/community#3114] [Help]
### Summary

I am trying to deploy a pretty barebones SvelteKit application and keep getting this error. I think it must have something to do with my vercel settings because my app builds & runs fine locally.

### Steps to Reproduce

Step 1: Create a skeleton application with `npm create svelte@latest my-app`
Step 2: Add it to a github repo
Step 3: Add the Vercel adapter and deploy

**Comment by swarnava** [comment]
Hey there,

Have you tried our Sveltekit boilerplate template: https://vercel.com/templates?search=sveltekit? This will probably help to get started.
